packages/react-components/src/components/kpi/kpi.tsx

```diff
@@ -5,7 +5,8 @@ import { useViewport } from '../../hooks/useViewport';
 import { widgetPropertiesFromInputs } from '../../common/widgetPropertiesFromInputs';
 import type { Annotations, StyleSettingsMap, Viewport, TimeSeriesDataQuery } from '@iot-app-kit/core';
 import type { KPISettings } from './types';
-import { DEFAULT_KPI_SETTINGS } from './constants';
+
+const DEFAULT_VIEWPORT: Viewport = { duration: '10m' };
```

Comment:
Non-blocking question: Why is the KPI not using a shared constant for default viewport which other visualizations would also use?

Reply:
done!